pubsub/pulsar/pulsar.go

```diff
@@ -92,7 +94,20 @@ func parsePulsarMetadata(meta pubsub.Metadata) (*pulsarMetadata, error) {
 		}
 		m.DisableBatching = disableBatching
 	}
-
+	if val, ok := meta.Properties[batchingMaxMessages]; ok {
```

Comment:
We don't have a default value here, and we rely on the upstream for it. So say if the default value of `bachingMaxMessages` get changed in the upstream, this will start using that on its own. 

However, if this value changes, our docs will be stale, since we are defining the default values there - https://github.com/dapr/docs/pull/2426/

Should we also hard code the defaults here to avoid that? Or instead of mentioning the default value in the docs, should we point to the upstream documentation? (I would prefer the former because it'll be less confusing for users). 

@saber-wang @daixiang0

Reply:
Hi, how can I format something like "5S" as "time. Duration"?

Reply:
Let's hardcode here and document it clearly.

Reply:
> And how can I format something like "5s" as "time. Duration"?

Sorry I did not understand this, can you explain more on what are you trying to do?

Reply:
@shubham1172  I have found `time.ParseDuration(val)` 😂.